redirects.yaml

```diff
@@ -23,6 +23,11 @@ redirects:
   - source: /subscribe
     destination: /newsletter
 
+  # Test post
+
+  - source: /test-post
+    destination: /handbook/content-types/example-post
```

Comment:
What's that redirect for?

Reply:
So the example post already existed. I've just moved it to the content-types directory (checked with Ewa beforehand) and thought it'd be better to redirect rather than leave a dead link. Might be unnecessary with it being a sample post though...

Reply:
I'd vote to remove 🙂 as it isn't publicly facing it shouldn't hurt and prevents someone from guessing why it's redirected in the first place